### PR TITLE
Add the new fragments for the pet info

### PR DESCRIPTION
**Pull request by AlbertPG**

### CODE REVIEW

**Review by xavier-campos-diaz — APPROVED** [review]

**Review by enrichp — APPROVED** [review]

**Comment by santidrj** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Issues
======
- Added 4
           

Complexity increasing per file
==============================
- app/src/main/java/org/pesmypetcare/mypetcare/activities/fragments/infopet/InfoPetFragmentAdapter.java  4
- app/src/main/java/org/pesmypetcare/mypetcare/activities/fragments/infopet/InfoPetMedicalProfile.java  1
- app/src/main/java/org/pesmypetcare/mypetcare/activities/fragments/infopet/InfoPetExercise.java  1
- app/src/main/java/org/pesmypetcare/mypetcare/activities/fragments/infopet/InfoPetVetVisits.java  1
- app/src/main/java/org/pesmypetcare/mypetcare/activities/fragments/infopet/InfoPetWashing.java  1
         
```


See the complete overview on [Codacy](https://app.codacy.com/gh/Grupo13-PES-Mascotas/PES_My-Pet-Care/pullRequest?prid=5464843&bid=17911236)

File: app/src/main/java/org/pesmypetcare/mypetcare/activities/fragments/infopet/InfoPetMedicalProfile.java [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+import android.view.View;
+import android.view.ViewGroup;
+
+import androidx.annotation.NonNull;
+import androidx.fragment.app.Fragment;
+
+import org.pesmypetcare.mypetcare.databinding.FragmentInfoPetMedicalProfileBinding;
+
+public class InfoPetMedicalProfile extends Fragment {
+    private FragmentInfoPetMedicalProfileBinding binding;
```
Comment (santidrj) [inline app/src/main/java/org/pesmypetcare/mypetcare/activities/fragments/infopet/InfoPetMedicalProfile.java:14]: Codacy Issue found: [Perhaps 'binding' could be replaced by a local variable.](https://app.codacy.com/gh/Grupo13-PES-Mascotas/PES_My-Pet-Care/pullRequest?prid=5464843)

File: app/src/main/java/org/pesmypetcare/mypetcare/activities/fragments/infopet/InfoPetVetVisits.java [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+import android.view.View;
+import android.view.ViewGroup;
+
+import androidx.annotation.NonNull;
+import androidx.fragment.app.Fragment;
+
+import org.pesmypetcare.mypetcare.databinding.FragmentInfoPetVetVisitsBinding;
+
+public class InfoPetVetVisits extends Fragment {
+    private FragmentInfoPetVetVisitsBinding binding;
```
Comment (santidrj) [inline app/src/main/java/org/pesmypetcare/mypetcare/activities/fragments/infopet/InfoPetVetVisits.java:14]: Codacy Issue found: [Perhaps 'binding' could be replaced by a local variable.](https://app.codacy.com/gh/Grupo13-PES-Mascotas/PES_My-Pet-Care/pullRequest?prid=5464843)

File: app/src/main/java/org/pesmypetcare/mypetcare/activities/fragments/infopet/InfoPetExercise.java [hunk 3]
```diff
@@ -0,0 +1,21 @@
...
+import android.view.View;
+import android.view.ViewGroup;
+
+import androidx.annotation.NonNull;
+import androidx.fragment.app.Fragment;
+
+import org.pesmypetcare.mypetcare.databinding.FragmentInfoPetExerciseBinding;
+
+public class InfoPetExercise extends Fragment {
+    private FragmentInfoPetExerciseBinding binding;
```
Comment (santidrj) [inline app/src/main/java/org/pesmypetcare/mypetcare/activities/fragments/infopet/InfoPetExercise.java:14]: Codacy Issue found: [Perhaps 'binding' could be replaced by a local variable.](https://app.codacy.com/gh/Grupo13-PES-Mascotas/PES_My-Pet-Care/pullRequest?prid=5464843)

File: app/src/main/java/org/pesmypetcare/mypetcare/activities/fragments/infopet/InfoPetWashing.java [hunk 4]
```diff
@@ -0,0 +1,21 @@
...
+import android.view.View;
+import android.view.ViewGroup;
+
+import androidx.annotation.NonNull;
+import androidx.fragment.app.Fragment;
+
+import org.pesmypetcare.mypetcare.databinding.FragmentInfoPetWashingBinding;
+
+public class InfoPetWashing extends Fragment {
+    private FragmentInfoPetWashingBinding binding;
```
Comment (santidrj) [inline app/src/main/java/org/pesmypetcare/mypetcare/activities/fragments/infopet/InfoPetWashing.java:14]: Codacy Issue found: [Perhaps 'binding' could be replaced by a local variable.](https://app.codacy.com/gh/Grupo13-PES-Mascotas/PES_My-Pet-Care/pullRequest?prid=5464843)

**Comment by santidrj** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Complexity increasing per file
==============================
- app/src/main/java/org/pesmypetcare/mypetcare/activities/fragments/infopet/InfoPetMedicalProfile.java  1
- app/src/main/java/org/pesmypetcare/mypetcare/activities/fragments/infopet/InfoPetFragmentAdapter.java  4
- app/src/main/java/org/pesmypetcare/mypetcare/activities/fragments/infopet/InfoPetVetVisits.java  1
- app/src/main/java/org/pesmypetcare/mypetcare/activities/fragments/infopet/InfoPetExercise.java  1
- app/src/main/java/org/pesmypetcare/mypetcare/activities/fragments/infopet/InfoPetWashing.java  1
         
```


See the complete overview on [Codacy](https://app.codacy.com/gh/Grupo13-PES-Mascotas/PES_My-Pet-Care/pullRequest?prid=5464843&bid=17911236)

**Pull request merged**